### PR TITLE
Add internal/config/client.go test

### PR DESCRIPTION
**Pull request by hlts2**
Signed-off-by: hlts2 <hiroto.funakoshi.hiroto@gmail.com>

### Description:

I added test for `internal/config/client.go` and godoc comment.

**gramer check passed**

### Environment:

- Go Version: 1.16
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.12.3

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [x] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [x] I have added tests and benchmarks to cover my changes.
- [x] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: internal/config/client_test.go [hunk 1]
```diff
@@ -47,37 +50,33 @@ func TestClient_Bind(t *testing.T) {
...
+		{
+			name: "return Client when the bind successes and net and transport is not nil",
+			fields: fields{
+				Net:       new(Net),
+				Transport: new(Transport),
+			},
+			want: want{
+				want: &Client{
+					Net:       new(Net).Bind(),
+					Transport: new(Transport).Bind(),
```
Comment (vankichi) [inline internal/config/client_test.go:68]: I think we do not use the same code that we'd like to test.

File: internal/config/client.go [hunk 2]
```diff
@@ -17,11 +17,13 @@
 // Package config providers configuration type and load configuration logic
 package config
 
+// Client represents Client configuration.
```
Comment (vankichi) [inline internal/config/client.go:20]:
```suggestion
// Client represents the Client configurations.
```

**Review by vankichi — DISMISSED** [review]

**Review by kevindiu — DISMISSED** [review]
LGTM

**Pull request merged**